### ISSUE TITLE
plex no media shows in add channel view

### ISSUE THREAD

**Issue by krohrsb** [vexorian/dizquetv#5]
After seemingly adding my plex server successfully I go to the channels section and click add channel. It shows my server name in the drop-down but no media is shown :(

Using docker setup

**Comment by killnfield** [comment]
I have the same issue no docker just using windows .exe.
I do have a huge library.
At this point I have tried to tweak a few things with no luck.

**Comment by killnfield** [comment]
Ok I think I have fixed that issue with no media. I disabled IP6 on my nic protocols and was able to see all my media

**Comment by dreamcasting** [comment]
I likewise have this problem. Running dizque on docker on my Synology NAS with Plex running outside of Docker.

**Comment by krohrsb** [comment]
I tested running the previous psuedotv itself and it seemed to work fine, just worth a note. I realize the server addition logic is different in dizuetv so perhaps something there.

**Comment by vexorian** [comment]
@krohrsb I bet you tested in 0.0.51 pseudotv. The new Plex authentication was added in Pseudotv 0.0.53

**Comment by dreamcasting** [comment]
It looks like there are two things possibly going on here.

To get Synology to work correctly with dizquetv:

(This assumes you are running dizquetv on Docker and Plex outside of it)

1. Stop the container
2. Click Edit
3. Go to "Environment"
4. Click the "+" to add a line
5. For variable put "add-host"
6. For value put the address shown under dizque's plex settings for the correct Plex server. Omit any "https://" or ":#####" port switches. It should be just the base url.
7. Add a ":" and the IP of the Plex server directly after what you pasted in the value field. In the end it should be something like "###-##-#-##.xxxxxxxx.plex.direct:###.#.##.###".
8. Click "Apply" 
9. Start the container

The second issue I'm seeing is there appears to be a small window of time where I can pull down my library and set a channel schedule but after a few minutes I can't pull it and Chrome's dev tools show the network traffic getting no response. I get a 401 when pinging the URL manually. Is there an authentication issue in play as well?

**Comment by blake1029384756** [comment]
I am seeing the same problem. I have both plex and dizquetv running as docker containers on the same host and docker network.

**Comment by hdiogosilva** [comment]
I was able to get around this problem by manually editing plex-servers.json.
Just changing the address & uri to my local address seems to do the trick.

I am also running plex and dizquetv as containers on the same host.

**Comment by krohrsb** [comment]
I also successfully got around this problem by modifying the plex-server.json file.

Its worth noting that these requests to plex happen in the browser. When running docker-based applications many things are abstracted/hidden and configured differently.

Knowing it was browser-based helped me choose the correct URI to connect to plex with.

**Comment by MoRBiD-MaNGLeR** [comment]
@krohrsb @dreamcasting @blake1029384756 @killnfield @vexorian

Should this Question/Bug be reworked into an Enhancement Request for making it easier to edit the address and URI in plex-servers.json?

Is there any other part of Issue #5 that needs to be addressed as an Issue/Bug, Question or Enhancement Request?  (If you have a new Issue/Bug, Question or Enhancement Request, please submit a new Issue with that information.)

**Comment by hcorn82** [comment]
Having the same problem even after manual editing the plex-servers.json and replacing all the address with my local IP of the plex server which is the same machine running dizquetv container. Maybe I am missing something...

**Comment by krohrsb** [comment]
> @krohrsb @dreamcasting @blake1029384756 @killnfield @vexorian
> 
> Should this Question/Bug be reworked into an Enhancement Request for making it easier to edit the address and URI in plex-servers.json?
> 
> Is there any other part of Issue #5 that needs to be addressed as an Issue/Bug, Question or Enhancement Request? (If you have a new Issue/Bug, Question or Enhancement Request, please submit a new Issue with that information.)

Likely yes, a feature request.

I now understand a bit better how things are working with dizquetv.

1. Uses plex auth, which retrieves a list of connections one can use. The main connection is automatically used in dizquetv.
1. The connection retrieved from plex may not be optimal in all situations (so yes, adding the ability to modify it would be useful)
1. I now know that the requests to plex happen Client (browser) side. This may dictate which url of plex people will use, depending on their setup.

**Comment by xinud190** [comment]
> I was able to get around this problem by manually editing plex-servers.json.
> Just changing the address & uri to my local address seems to do the trick.
> 
> I am also running plex and dizquetv as containers on the same host.

Can anyone provide an example of an altered file?  I've tried making changes and can't seem to get it to work.

No content appearing when trying to add programming.

Can't figure it out.

**Comment by vexorian** [comment]
Hello all. Work for version 63 is focusing in making it easier to deal with this problem. It will be possible to edit the server values from the web UI instead of touching a json. And you won't have to remake the channels after editing them. Then in 64 it will be easier to configure this stuff correctly.

**Comment by blake1029384756** [comment]
When I made my changes the ip of plex in my docker network didn't work. I had to port forward to the host and use its ip.

**Comment by xinud190** [comment]
Ok great, I will wait on future versions and retry.  Thanks for the response.

Project looks promising - thank you!

**Comment by vexorian** [comment]
I haven't seen this reported in a while, I think the server stuff in 0.0.63+ has actually made people more aware of when their plex can't be accessed in  the UI. Closing it for the time being.
